### DISCUSSION TITLE
[Feature] Hot key to turn outliner on/off

### DISCUSSION THREAD

**Discussion by dammitcoetzee** [vslinko/obsidian-outliner#181] [Ideas]
Coming from Roam 90% of the time I am working in bullet lists and love outliner, but sometimes I want to edit the page normally or just fix some messy bullets which are confusing outliner from a copy/paste. As far as I can tell there is no way to quickly enable/disable the formatting/editing features of outliner. Would this be possible?

Thanks,
Gerrit

**Comment by Cito** [comment]
As a workaround, you can configure a keyboard shortcut that opens an external editor like Typora for this and similar cases where the built-in editor or plugins do not work very well.

**Reply by vslinko** [reply]
@dammitcoetzee Hello! Thank you for the idea. It is possible to implement it. I will implement it in the future.

**Reply by dammitcoetzee** [reply]
Thanks so much for the plugin and the reply. I look forward to this functionality one day in the future, and I appreciate you adding it to the list.

**Comment by jpmoo** [comment]
I’d like to second this kind of functionality. I’d love to work on daily notes and some others with Outliner, but turn it off for evergreen and other notes. Ideally, I could leave it off and then turn it on through YAML in certain templates, or specify certain folders to use it or not use it.

**Reply by ghost** [reply]
"Ideally, I could leave it off and then turn it on through YAML in certain templates, or specify certain folders to use it or not use it."

THIS!
I'd love to have Outliner always enabled in my Daily Notes folder, and always disabled in my other folders.
